docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -44,8 +51,66 @@ The following are the required environment variables with some example values:
 
 ```shell
 # Your standard region that you host AWS resources in
-export REGION=eu-central-1
-# Following two environment variables can be skipped if you don't have a domain
+export REGION="$AWS_REGION"
+
+# The Camunda 8 Helm Chart version
+export CAMUNDA_HELM_CHART_VERSION="11.0.0"
+```
+
+<Tabs groupId="env">
+
+  <TabItem value="basic" label="Standard" default>
```

Comment:
just a suggestion to keep it consistent with your other doc where you used `value="standard" label="Standard"`